README.rst

```diff
@@ -40,33 +40,58 @@ Dependencies
 ------------
 The following libraries are used in building this code:
 
-- libxc (from Xen, the Xen Control library, required for Xen support)
+- ``autotools``
 
-- libxenstore (from Xen, access to the xenstore, required for Xen support)
+- ``libtool`` Generic library support script
 
-- libvirt (from Red Hat, access to KVM guests, 0.8.7 or newer required for KVM
-  support, MUST BE BUILT WITH QMP SUPPORT -- THIS REQUIRES yajl)
+- ``check`` Unit test framework for C
 
-- qemu-kvm patch (option 1 for KVM memory access, optional for KVM support,
-  still buggy but faster than alternative option, see Note 2)
+- ``yacc`` OR ``bison`` (for reading the configuration file)
 
-- gdb enabled kvm VM (option 2 for KVM memory access, optional for KVM
-  support, more stable than option 1 but slower, see Note 2)
+- ``lex`` OR ``flex`` (for reading the configuration file)
+
+- ``glib`` (``>= 2.22``)
 
-- yacc OR bison (for reading the configuration file)
+Installing the dependencies on Ubuntu::
 
-- lex OR flex (for reading the configuration file)
+    $ sudo apt-get install autotools-dev automake check flex bison libglib2.0-dev
 
-- glib (version 2.22 or better is required)
+Xen support
+~~~~~~~~~~~
 
-Note 1: If you are installing a packaged version of Xen, you will likely
+- ``libxc`` (from Xen, the Xen Control library, required for Xen support)
+
+- ``libxenstore`` (from Xen, access to the xenstore, required for Xen support)
```

Comment:
If we are updating the README, might as well change this to say that libxenstore is actually optional.